server/public/index.html

```diff
@@ -16,11 +16,13 @@
     href="https://fonts.googleapis.com/css2?family=Jura:wght@300;400;500;600;700&family=Orbitron:wght@400;500;600;700;900&display=swap"
     rel="stylesheet">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+
 </head>
 
 <body>
   <div id="root"></div>
   <script src="main.js"></script>
+  <script src="/socket.io/socket.io.js"></script>
```

Comment:
You should use npm to install the `socket.io-client` and `import` it into your code.